src/utils/ReskindexPlugin.js

```diff
@@ -0,0 +1,71 @@
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+
+function ReskindexPlugin(header) {
```

Comment:
please make sure you add a commend explaining what this does and what the parameter is